bliss/encoder/metrics.py

```diff
@@ -138,36 +138,31 @@ def get_report_bins(self):
 
 
 class FluxBinMetric(GeneralBinMetric):
-    def __init__(self, base_nmgy_bin_cutoffs: list, report_bin_unit: str, exclude_last_bin: bool):
-        super().__init__(bin_cutoffs=base_nmgy_bin_cutoffs, exclude_last_bin=exclude_last_bin)
+    def __init__(self, base_njy_bin_cutoffs: list, report_bin_unit: str, exclude_last_bin: bool):
+        super().__init__(bin_cutoffs=base_njy_bin_cutoffs, exclude_last_bin=exclude_last_bin)
 
-        self.base_nmgy_bin_cutoffs = self.bin_cutoffs
+        self.base_njy_bin_cutoffs = self.bin_cutoffs
         self.report_bin_unit = report_bin_unit
-        assert self.report_bin_unit in {"mag", "nmgy", "njymag"}, "invalid bin type"
+        assert self.report_bin_unit in {"njy", "ab_mag"}, "invalid bin type"
 
     def get_state_for_report(self, state_name):
         state = getattr(self, state_name)
         match self.report_bin_unit:
-            case "nmgy":
+            case "njy":
```

Comment:
I don't use njy for SDSS, only nmgy

Reply:
I wish there was a generic term that could mean either

Reply:
change it to `flux`